awx/main/tests/functional/api/test_job_runtime_params.py

```diff
@@ -157,11 +189,28 @@ def test_job_accept_empty_tags(job_template_prompts, post, admin_user, mocker):
         with mocker.patch('awx.api.serializers.JobSerializer.to_representation'):
             post(reverse('api:job_template_launch', kwargs={'pk': job_template.pk}), {'job_tags': '', 'skip_tags': ''}, admin_user, expect=201)
             assert JobTemplate.create_unified_job.called
-            assert JobTemplate.create_unified_job.call_args == ({'job_tags': '', 'skip_tags': ''},)
+            assert JobTemplate.create_unified_job.call_args == ({'job_tags': '', 'skip_tags': '', 'forks': 1, 'job_slice_count': 0},)
```

Comment:
@john-westcott-iv going through tests, I was surprised by these numbers. This doesn't look desirable, I would prefer these files to default to `None` / not-provided.

Reply:
addressed in https://github.com/ansible/awx/pull/12917